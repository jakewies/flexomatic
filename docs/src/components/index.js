export { default as Section } from './Section'
export { UnorderedList, ListItem } from './List'
export { Header1, Header2, TextItem, TextBlock, Code, Pre } from './Typography'
export { default as FancyLink } from './FancyLink'
export { default as Example } from './Example'
export { default as Page } from './Page'
