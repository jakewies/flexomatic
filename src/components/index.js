export { default as Grid } from './Grid'
export { default as Cell } from './Cell'
export { default as Demo } from './Demo'
