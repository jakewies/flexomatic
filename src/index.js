export { Grid, Cell, Demo } from './components'
