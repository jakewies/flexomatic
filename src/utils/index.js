export { default as align } from './align'
