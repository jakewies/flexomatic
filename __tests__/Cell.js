import React from 'react'
import { Cell } from '../src/'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

describe('Cell', () => {
  it('renders', () => {
    const tree = renderer.create(<Cell>Test Cell</Cell>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
