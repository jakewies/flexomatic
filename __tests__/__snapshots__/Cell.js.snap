// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cell renders 1`] = `
.c0 {
  box-sizing: border-box;
  padding: 1em 0 0 1em;
  display: block;
  -webkit-align-self: stretch;
  -ms-flex-item-align: stretch;
  align-self: stretch;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

<div
  className="c0"
>
  Test Cell
</div>
`;
